parameter-sets/node-pool-request/parameter-set.json

```diff
@@ -99,12 +99,40 @@
             "type":"SEPARATOR",
             "label" :"Advanced options"
         },
+        {
+            "name": "nodepoolLabels",
+            "type": "MAP",
+            "label": "Labels",
+            "description": "Annotations specific to this node pool"
+        },
         {
             "name": "serviceAccount",
```

Comment:
Is there a risk for existing clusters ?

Reply:
There was a bit of confusion here :) At the node pool level, "labels" are [K8S labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/) whereas at the cluster level, "labels" are resourceLabels, ie simple tags that GCP puts on the VMs. 
Adding custom K8S labels on node pools has its uses, but if it introduces useless complexity I'm OK with removing it, especially if we think about introducing node taints in the future.

Reply:
Fixed.